processor/transformer/transformer.go

```diff
@@ -310,6 +315,39 @@ func (t *HTTPLBTransport) NewRoundTripper(scheme, target string, config httplb.T
 	return httplb.RoundTripperResult{RoundTripper: t.Transport, Close: t.CloseIdleConnections}
 }
 
+func (trans *handle) getPicker() func(prev picker.Picker, allConns conn.Conns) picker.Picker {
+	pickerType := config.GetString("Transformer.Client.httplb.pickerType", "power_of_two")
+	switch pickerType {
+	case "power_of_two":
+		return picker.NewPowerOfTwo
+	case "round_robin":
+		return picker.NewRoundRobin
+	case "least_loaded_random":
+		return picker.NewLeastLoadedRandom
+	case "least_loaded_round_robin":
+		return picker.NewLeastLoadedRoundRobin
+	case "random":
+		return picker.NewRandom
+	default:
+		panic(fmt.Sprintf("unknown picker type: %s", pickerType))
+	}
+}
+
+func (trans *handle) getChecker() health.Checker {
+	checkerType := config.GetString("Transformer.Client.httplb.checkerType", "nop")
+	switch checkerType {
+	case "nop":
+		return health.NopChecker
+	case "polling":
+		return health.NewPollingChecker(
+			health.PollingCheckerConfig{},
+			health.NewSimpleProber(trans.config.userTransformationURL),
```

Comment:
wouldn't using `trans.config.userTransformationURL` adversely affect destination transformation considering the same client is used..?

Reply:
I guess there's no adverse effects, but we're not probing the right endpoint

Reply:
Will be fixing this in the new separate clients PR